File: linguagens/Python/Exercicios/retangulo.py
base: float
altura: float 
area: int
perimetro: int
diagonal: float

base = float(input("Insira a base do Triangulo: "))
altura = float(input("Insira a altura do Triangulo: "))

area = (base * altura)
perimetro = base*2 + altura*2

diagonal = (base**2 + altura**2)**(1/2) 

print(f"Area: {area:.4f}")

print(f"Perimentro: {perimetro:.4f}")

print(f"diagonal: {diagonal:.4f}")

File: linguagens/C++/exercicios/retangulo.cpp
#include <iostream> 
#include <cmath> 

using namespace std; 

int main()
{
    float base, altura, area, perimetro, diagonal; 

    cout << "Base do retangulo: ";  
    cin >> base; 
    cout << "Altura do retangulo: ";  
    cin >> altura; 
    
    area = base * altura ; 
    perimetro = 2 * (base + altura);
    diagonal =  sqrt((altura*altura) + (base*base)); 

    cout << "Area: " << area << endl; 
    cout << "Perimentro: " << perimetro << endl; 
    cout << "Diagonal: " << diagonal << endl; 
    




    return 0; 
}